k8s/examples/wordpress/wordpress/19-pvc.yaml.DIS

```diff
@@ -0,0 +1,13 @@
+kind: PersistentVolumeClaim
```

Comment:
How does this PVC get used? There's a volumeClaimTemplate in the statefulset so does that make this file redundant?

Reply:
forgot to removed it!